webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -11,24 +12,23 @@ import { manifestExists } from '../SubscriptionHelpers';
 import { columns } from './ManifestHistoryTableSchema';
 import DeleteManifestModalText from './DeleteManifestModalText';
 
+export const MANAGE_MANIFEST_MODAL_ID = 'manageManifestModal';
+export const DELETE_MANIFEST_MODAL_ID = 'deleteManifestModal';
```

Comment:
This should be in the consts file

Reply:
I thought this made more sense here, in the same file that uses `ForemanModal`, because
* The constants file is customarily used for standardizing the names of Redux actions, but the ForemanModal ID is not an action name
* You could theoretically have a component that doesn't use any Redux actions but still has a ForemanModal ID, and is controlled from elsewhere
* Brings the value closer to where it is used.

I can also see the other side of the argument, which is "all constant string values should be organized in the same file."  But I'm partial to keeping it in the same file for the reasons above.  Thoughts?

Reply:
I think the consts file is for all consts for example we also use it for `STATUS` which is not a Redux action but a value that we use.

> Brings the value closer to where it is used.

I saw that the ID was imported in another file so its not close to it :)

Reply:
@MariaAga ok fine, I'll move them :yum: